lablink_sleap_client_image/Dockerfile

```diff
@@ -0,0 +1,94 @@
+# Base image with SLEAP and GPU support
+# https://hub.docker.com/repository/docker/eberrigan/sleap-cuda/general
+FROM eberrigan/sleap-cuda:latest
```

Comment:
let's change this to the ghcr `FROM ghcr.io/talmolab/sleap-cuda:linux-amd64-2a8cefd7d0291d2ce07998fa3c54c9d0d618d31b`
double check that is the right image https://github.com/talmolab/sleap-cuda-container/pkgs/container/sleap-cuda/345206219?tag=linux-amd64-2a8cefd7d0291d2ce07998fa3c54c9d0d618d31b